sway/tree/container.c

```diff
@@ -181,12 +362,39 @@ struct sway_container *container_output_create(
 	return output;
 }
 
-struct sway_container *container_workspace_create(
-		struct sway_container *output, const char *name) {
-	if (!sway_assert(output,
-			"container_workspace_create called with null output")) {
-		return NULL;
+static struct sway_container *get_workspace_initial_output(const char *name) {
+	struct sway_container *parent;
+	// Search for workspace<->output pair
+	int i, e = config->workspace_outputs->length;
+	for (i = 0; i < e; ++i) {
+		struct workspace_output *wso = config->workspace_outputs->items[i];
+		if (strcasecmp(wso->workspace, name) == 0) {
+			// Find output to use if it exists
+			e = root_container.children->length;
```

Comment:
Eh, `e` is changed inside the loop but is used outside. Even if it works because there's a `break`, it's very confusing

Reply:
It's not related to closing, so I'll do it in another PR.